### ISSUE TITLE
Openlayers export and rasters/ projected wrong

### ISSUE THREAD

**Issue by riccardoklinger** [tomchadwin/qgis2web#33]
![screenshot from 2015-05-01 23 08 22](https://cloud.githubusercontent.com/assets/7377897/7437345/097ca33a-f057-11e4-928a-95ae8df3e4cb.png)
rasters are misplaced/distorted in export

**Comment by tomchadwin** [comment]
@volaya Does qgis-ol3 warp rasters to fit the CRS (either via native QGIS functionality or via GDAL)? If not, we need to call the qgis2leaf code for that. If it does, we need to see what difference there is between qgis-ol3 and qgis2web.

**Comment by tomchadwin** [comment]
I've copied over qgis2leaf's raster warp code. However, when qgis2leaf adds the raster to the map, it seems to warp its extents in the client:

```javascript
var img_n3311314284= 'data/exp_n3311314284.png';
var img_bounds_n3311314284 = [[52.8104851578,13.9696021283],[53.0217322197,14.2834529158]];
var overlay_n3311314284 = new L.imageOverlay(img_n3311314284, img_bounds_n3311314284).addTo(map);
```

I don't know what is required for OL3. Does anyone else know?

**Comment by lucacasagrande** [comment]
Hi,
I have tried to comment out [imageSize](https://github.com/tomchadwin/qgis2web/blob/master/olwriter.py#L443) and:

- The raster is exported and shown in the Preview;
- It's in the correct position;

I'll do further tests.

**Comment by tomchadwin** [comment]
Looks good to me - fantastic! Are you happy for me to close this issue, @lucacasagrande?

**Comment by tomchadwin** [comment]
Released as 1.0.0-RC3 and pushed to QGIS plugins repo
